msal/tests/Test.MSAL.NET.UIAutomation/XamarinDroidMSALTests.cs

```diff
@@ -64,14 +67,42 @@ public void InitializeBeforeTest()
         }
 
         /// <summary>
-        /// Runs through the standard acquire token flow
+        /// Runs through the standard acquire token flow, using the default app configured UiBehavior = Login
         /// </summary>
         [Test]
         public void AcquireTokenTest()
         {
             _msalMobileTestHelper.AcquireTokenInteractiveTestHelper(xamarinController, LabUserHelper.GetLabResponseWithDefaultUser());
         }
 
+        /// <summary>
+        /// Runs through the standard acquire token flow
+        /// </summary>
+        [Test]
+        public void PromptBehavior_Consent_SelectAccount()
```

Comment:
We are looking to add iOS tests soon. thus, there will most likely be a XamariniOSMSALTests class with the required parameters for iOS testing. It would be great if we can share this functionality between both platforms. My original idea was to create a test helper that contained the core functionality of the mobile tests and use the XamarinDroidMSALTests and XamariniOSMSALTests classes to pass in the parameters needed for the test on those platforms(which will be the same in most cases). Any thoughts on how we can share this functionality with the iOS equivalent test?